src/searchAndReplace.ts

```diff
@@ -54,7 +54,7 @@ interface TextNodesWithPosition {
 }
 
 const getRegex = (s: string, disableRegex: boolean, caseSensitive: boolean): RegExp => {
-  return RegExp(disableRegex ? s.replace(/[-/\\^$*+?.()|[\]{}]/g, "\\$&") : s, caseSensitive ? "gu" : "gui")
+  return RegExp(disableRegex ? s.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') : s, caseSensitive ? "gu" : "gui");
```

Comment:
taken from [this comment](https://github.com/sereneinserenade/tiptap-search-and-replace/issues/4#issuecomment-1887197113)